juniper/src/parser/tests/document.rs

```diff
@@ -145,3 +146,20 @@ fn errors() {
         )
     );
 }
+
+#[test]
+fn issue_427_panic_is_not_expected() {
+    struct QueryWithoutFloat;
+
+    #[crate::object_internal]
+    impl QueryWithoutFloat {
+        fn get_int(value: String) -> String {
+            value
+        }
+    }
+
+    let schema = SchemaType::new::<QueryWithoutFloat, EmptyMutation<()>>(&(), &());
+    let parse_result = parse_document_source(r##"{ getInt(value: 123.0) }"##, &schema);
+
+    assert!(parse_result.is_err());
```

Comment:
We should check that it is an `ExpectedScalarError` rather than just check for any error to prevent shadowing a possible underlying bug.

Reply:
Done